pkg/rancher-desktop/window/index.ts

```diff
@@ -222,18 +222,16 @@ const updateView = (window: any, payload: any) => {
   }
 
   const contentSize = window.getContentSize();
-  const titleBarHeight = 0;
-
   const yZoomFactor = window.webContents.getZoomFactor();
-
   const x = Math.round(payload.x * window.webContents.getZoomFactor());
-  const y = Math.round((payload.y + titleBarHeight) * yZoomFactor);
+  const y = Math.round(payload.y * yZoomFactor);
+  const footerHeight = 25;
```

Comment:
The footer height needs to take zoom into account too.
![image](https://github.com/rancher-sandbox/rancher-desktop/assets/3977982/8cc4dbf0-9935-4720-aad8-51d428eb27ce)

Reply:
That's right. I'm not sure why I opted to treat it as a constant value.. ready for another review.